core/lib/spree/core/controller_helpers/strong_parameters.rb

```diff
@@ -10,6 +10,10 @@ def permitted_attributes
                  to: :permitted_attributes,
                  prefix: :permitted
 
+        def permitted_credit_card_attributes
+          permitted_source_attributes
```

Comment:
We can't use this for user facing authorization.

The user can change the following:

```
[:number,
 :month,
 :year,
 :expiry,
 :verification_value,
 :first_name,
 :last_name,
 :cc_type,
 :gateway_customer_profile_id,
 :gateway_payment_profile_id,
 :last_digits,
 :name,
 :encrypted_data]
```